### PR TITLE
Replace simplejson with python json in openlibrary/data package for 2308

### PR DESCRIPTION
**Pull request by dherbst**
Addresses #2308

Refactor to replace simplejson with python json in openlibrary/data package

### Technical

Refactor to replace simplejson with python json in openlibrary/data package

### Testing

Unit tests pass.

### Screenshot

N/A

### Stakeholders

@cclauss

### CODE REVIEW

File: openlibrary/data/__init__.py [hunk 1]
```diff
@@ -9,7 +9,7 @@ def parse_data_table(filename):
     <key, type, revision, json> for all entries.
     """
     for line in open(filename):
-        thing_id, revision, json = pgdecode(line).strip().split("\t")
-        d = simplejson.loads(json)
+        thing_id, revision, json_data = pgdecode(line).strip().split("\t")
+        d = json.loads(json_data)
         yield d['key'], d['type']['key'], str(d['revision']), json
```
Comment (mekarpeles) [inline openlibrary/data/__init__.py:14]: BLOCKER: I think this is wrong -- this line now references `json` the library, not `json_data`

Comment (dherbst) [reply openlibrary/data/__init__.py]: Good catch!

**Comment by dherbst** [comment]
I also resolved the 2 lint warnings as well.  

Ready for review, please.

**Comment by cclauss** [comment]
OK...  So you both understand the general approach on this series of PRs.  We will not land them all at once.

Instead, we will land the test and mock PRs first.  After that, we will land one or two PRs a week so they do not all go into a single production deployment.  There are subtle differences between the two json implementations so just because our tests pass does not mean that we will have identical code before/after each of these changes.

@dherbst If you could find any docs on when this change might cause troubles, that would be very helpful.  Thanks much for doing this 👍 and for your patience yet persistence as we land these into the codebase.

**Comment by dherbst** [comment]
The plan makes sense to me.  Thanks for taking a look at the PRs.

> If you could find any docs on when this change might cause troubles, that would be very helpful. Thanks much for doing this 👍 and for your patience yet persistence as we land these into the codebase.

Will do.  I'll put the info into the issue #4303 so it should be easy to find later.

**Review by cclauss — APPROVED** [review]

**Pull request merged**